app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -228,6 +233,12 @@ export default {
       queriesPromises.push(this.getPrivateQueries(userId));
     }
 
+    if (!userId || userId.length === 0) {
+      this.isUserLogged = false
+    } else {
+      this.isUserLogged = true
+    }
```

Comment:
Why do you do a negative clause? I think it's simpler:

```suggestion
      this.isUserLogged = userId && userId.length
```
FYI
https://scotch.io/bar-talk/5-tips-to-write-better-conditionals-in-javascript

Reply:
I've done even a performance test https://jsben.ch/AAi5F, the difference are not so remarkable but it wins positive clauses

Reply:
we need to check if user is not logged, in this case the API return an empty string.

`this.isUserLogged = userId && userId.length`
return a `number`, we need a `Boolean`.

Reply:
Hahah I love Javascript, it's almost a language.

Why not `this.isUserLogged = (!userId || userId.length === 0)` ? Just asking

Reply:
`this.isUserLogged = (!userId || userId.length === 0)` Return a `Number` if user is logged in, and return `Boolean` if user is not logged

`this.isUserLogged = (userId.length > 0)` always return a `Boolean`

Reply:
If you do something like this `!!(!userId || userId.length === 0)` or `!!(userId && userId.length)` you enforce the Boolean result

https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054